opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -1,5 +1,5 @@
 
-
+#include <opm/simulators/DeferredLoggingErrorHelpers.hpp>
```

Comment:
Seems this file is missing a copyright header, please add one...

Reply:
done